test/test_projects_api.py

```diff
@@ -70,11 +70,11 @@ def test_retrieve_projects(self):
             self.assertGreater(project.average_price_per_tonne_cents_usd, 0)
             self.assertGreater(project.remaining_mass_g, 0)
             self.assertEqual(project.standard, None)
-            self.assertRegex(project.name, r"Test Offset Project")
+            self.assertIsInstance(project.name, str)
             self.assertTrue(project.description)
-            self.assertEqual(project.country, "US")
-            self.assertEqual(project.type, "biomass")
-            self.assertEqual(project.developer, "Patch Sandbox Supplier Test")
+            self.assertIsInstance(project.country, str)
+            self.assertIsInstance(project.type, str)
+            self.assertIsInstance(project.developer, str)
```

Comment:
no test for verifier?

Reply:
Unfortunately, that data was too sparsely populated to make for a reliable test. To be revised when we can improve test strategies for SDKs.